src/components/inputs/money-input/money-input.js

```diff
@@ -502,12 +527,18 @@ class MoneyInput extends React.Component {
           }}
         >
           <Select
-            inputId={MoneyInput.getCurrencyDropdownId(this.props.id)}
+            inputId={id}
             name={getCurrencyDropdownName(this.props.name)}
             value={option}
-            isDisabled={this.props.isDisabled || hasNoCurrencies}
+            isDisabled={
+              this.props.isDisabled || hasNoCurrencies || this.props.isReadOnly
```

Comment:
Should the select rather also be in readonly mode if this component is in it? Feels a bit odd to mix states in that readonly in one becomes disabled on another.

Reply:
@tdeekens react-select doesn't seem to have a `readonly` prop [ref](https://react-select.com/props)